vasyl_varyshchuk/Week_3/google_search_request.py

```diff
@@ -0,0 +1,46 @@
+"""
+This module sends a request to google.com returning the response headers
+and the specified number of searching results
+"""
+import requests
+from bs4 import BeautifulSoup
```

Comment:
Is the `bs4` a default library?

Reply:
No, it isn't, Should I use only standard libraries?

Reply:
You should:
1) add some message
2) add a handler for import non-default libraries
3) Add an `requirements.txt` file with a needed dependency(read more about this part)

Reply:
I added an ability to install the required module from requirements.txt file and committed my changes.